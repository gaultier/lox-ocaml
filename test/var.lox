var a = 5*2; var b =20; var c= b /2;

print a; print b;
print c;

print a == c;
