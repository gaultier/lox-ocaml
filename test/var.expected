10.000000
20.000000
10.000000
true
